### DISCUSSION TITLE
Markdown support?

### DISCUSSION THREAD

**Discussion by justingoldberg** [damus-io/damus#252] [Ideas]
Markdown support would be a good idea.

**Comment by BenGWeeks** [comment]
It is? (Although a limited subset)

**Reply by ghost** [reply]
Hi BenGWeeks.

> It is? (Although a limited subset)

Markdown is a lightweight markup language for creating formatted text using a plain-text editor. John Gruber and Aaron Swartz created Markdown in 2004.

**Comment by hodlrodeo** [comment]
I found markdown to be a good way to store my long form text, it's easy to translate out of for other purposes. It was simple to post long form to YakiHonne app with markdown
